design/open/spark-outputcommitter/committer.md

```diff
@@ -187,8 +211,17 @@ the same paths but keeping things safe regardless.
 [^2]: 	Whenever there is a conflict, we want the task branch (which will become
     the "latest writer" after a successful HOC commit) to win.
 
+# Potential wins
+
+* Explicit requests _not_ to write 3 times and give better behaviour than
+  the default FileOutputCommitter from Spark have appeared on our Slack
+  [#data-architecture-discussion][slack-dont-write-thrice].
+* Multiple users have requested "overwrite" save mode.
+* Multiple users have requested multi-writer support.
+* Note by a developer that Spark performs many lakeFS API operations when
```

Comment:
and also it makes things more predictable from the lakeFS server perspective. with FOC we know what amount of API calls to expect and their types.